src/locales/ar/common.json

```diff
@@ -3466,7 +3466,6 @@
   "Terms": {
     "title": "شروط الاستخدام",
     "read": "قراءة شروط الاستخدام",
-    "switchLabelPrivacy": "لقد قرأت ووافقت على <1>سياسة الخصوصية</1>",
```

Comment:
Once you remove from english crowdin, our collaborative translation platform should handle the rest :smile_cat: 
this is not necessary but no worries we can keep the removal anyways.

Reply:
oh I see, good to know!